projects/js-packages/licensing/components/activation-screen-controls/style.scss

```diff
@@ -0,0 +1,108 @@
+@import '../rna-styles';
+
+.jp-license-activation-screen-controls {
+	background: var( --jp-white );
+	padding: 64px;
+	display: flex;
+	flex-direction: column;
+	justify-content: space-between;
+
+	h1 {
+		font-size: var( --font-title-large );
+		font-weight: 700;
+	}
+
+	p {
+		font-size: var( --font-body );
+	}
+
+	label {
+		font-weight: 600;
+		font-size: var( --font-body );
+	}
+}
+
+.jp-license-activation-screen-controls--license-field,
+.jp-license-activation-screen-controls--license-field-with-error {
+	label {
+		font-weight: 600;
+		font-size: var( --font-body );
+	}
+
+	input.components-text-control__input {
+		border-radius: var( --jp-border-radius );
+		border: 1px solid #787c82;
+		font-size: 18px;
+		line-height: 24px;
+		margin: 0;
+		min-height: 48px;
+	}
+}
+
+.jp-license-activation-screen-controls--license-field {
+	input.components-text-control__input {
+		border: 1px solid #787c82;
+	}
+}
+
+.jp-license-activation-screen-controls--license-field-with-error {
+	input.components-text-control__input {
+		border: 1px solid var( --jp-red );
+	}
+}
+
+.jp-license-activation-screen-controls--license-field-error {
+	display: flex;
+	flex-direction: row;
+	align-items: center;
+	color: var( --jp-red );
+
+	.error-gridicon {
+		margin-right: 4px;
+		fill: var( --jp-red );
+	}
+
+	span {
+		font-size: var( --font-body );
+	}
+}
+
+.jp-license-activation-screen-controls--button {
+	font-size: var( --font-body );
+
+	background-color: var( --jp-green-primary );
+	border-color: var( --jp-green-primary );
+	border-radius: var( --jp-border-radius );
+	color: var( --jp-white );
+
+	padding: 13.5px 45px 13.5px 45px;
+	min-height: 48px;
+
+	margin-top: 24px;
+
+	&:hover {
+		background-color: #007117;
+		border-color: #007117;
+		color: var( --jp-white );
+	}
+
+	&:focus {
+		background-color: #005b18;
+		border-color: var( --jp-white );
+		box-shadow: 0 0 0 1px #005b18;
+		color: var( --jp-white );
+	}
+
+	&:active {
+		background: #007117;
+		border-color: #007117;
+		color: var( --jp-white );
+	}
+
+	&[disabled],
+	&:disabled {
+		background: #dcdcde;
```

Comment:
I think we replace `#dcdcde` with `var( --jp-gray )`.

See https://github.com/Automattic/jetpack/blob/59940645adf7ee643b1efb71a866a4f93d2c68b2/projects/js-packages/licensing/components/rna-styles.scss#L14